### DISCUSSION TITLE
pack: extending runtime base image

### DISCUSSION THREAD

**Discussion by sefini** [buildpacks/community#242] [Q&A] [answered]
hi all,

Going through the [run-dockerfile-extend](https://buildpacks.io/docs/extension-guide/create-extension/run-dockerfile-extend/) doc.  While trying to `pack build`, using the [cowsay extension sample code](https://github.com/buildpacks/samples/blob/main/extensions/cowsay/bin/generate), got the error `run.Dockerfile should not expect arguments  Error: failed to build: executing lifecycle: failed with status code: 92`.    I should mention, before this error, i got the `(Buildpack API 0.10) is incompatible with lifecycle 0.16.5 (Buildpack API(s) 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9)` error.  Which is based on the [sample code here](https://github.com/buildpacks/samples/blob/main/extensions/cowsay/extension.toml#L2).  Got passed it by using version .9.  

Pack cli version: `0.31.0+git-3a994bd.build-5086`

**Comment by natalieparellano** [comment, marked as answer]
@sefini it appears that the lifecycle version here is too old. Can you try pull the latest version of the samples repo?

**Reply by sefini** [reply]
thank u
